package.json

```diff
@@ -19,6 +19,7 @@
     "@emotion/styled": "^11.8.1",
     "@mui/icons-material": "^5.3.1",
     "@mui/material": "^5.3.0",
+    "@polkadot/keyring": "^8.4.1",
```

Comment:
shouldn't need to specify this in package.json as it is a part of `@cennznet/api`